importer/yarn_to_gd.gd

```diff
@@ -45,21 +45,31 @@ static func command(command, parameters):
 					"duration": float(parameters[0])
 				})
 			]
+		"set":
```

Comment:
According to the [Yarn docs](https://yarnspinner.dev/docs/unity/working-with-commands/), the only two built-in commands available from the Yarn implementation are `stop` and `wait`, while other commands are implemented by the game, so I may remove this to be syntax-compliant :sweat_smile:

Reply:
Totally fair. I got this syntax from the [Yarn docs variables](https://yarnspinner.dev/docs/writing/expressions-and-variables/#variables) page. I'm not sure if this is just the documentation being inconsistent. Since it stated `To set the value of a variable, you use the set keyword.` I took this to mean it was a built-in command. This seems a bit like a contradiction in the source spect to me. Hmm... :/

It's fairly trivial to wire up something to set variables... and possibly more consistent. So I'm cool with ditching it if you think that's best. ¯\_(ツ)_/¯

Reply:
Oh that's a good point too. I might just ask the Yarn Devs to see what they consider the formal spec xD

Reply:
Confirmed it, it's part of the spec but not really a command. So I'll keep it in :P